libafl/src/corpus/testcase.rs

```diff
@@ -344,6 +360,21 @@ impl SchedulerTestcaseMetaData {
 
 crate::impl_serdeany!(SchedulerTestcaseMetaData);
 
+#[cfg(feature = "std")]
+impl<I> Drop for Testcase<I>
+where
+    I: Input,
+{
+    fn drop(&mut self) {
+        if let Some(filename) = &self.filename {
+            let mut path = PathBuf::from(filename);
+            let lockname = format!(".{}.lafl_lock", path.file_name().unwrap().to_str().unwrap());
+            path.set_file_name(lockname);
+            std::fs::remove_file(path).ok();
```

Comment:
Why `.ok()` and not just assign this to an underscore, or call `drop` on the value? I needed to google what does actually does :D

Reply:
I normally use .ok() instead of let _ bindings :slightly_smiling_face: I can swap it out